### PR TITLE
Add subheadings and edit introduction

### PR DESCRIPTION
**Pull request by bryanhce**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: User Guide
 GuestBook is a **desktop app for managing guests in a hotel,
 optimized for use via a Command Line Interface** (CLI)
 while still having the benefits of a Graphical User Interface (GUI).
-If you can type fast, GuestBook can get your
+As a hotel manager, if you can type fast, GuestBook can get your
```
Comment (JustinPeng13) [inline docs/UserGuide.md]: Would "If you are a hotel manager who can type fast" be a better phrasing?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -171,15 +173,15 @@ Constraints of parameters:
 * `NUMBER_OF_GUESTS` should only contain numbers, and it should be at least 1 and at most 4.
```
Comment (JustinPeng13) [inline docs/UserGuide.md:186]: For email above, can we say "local segment" and "domain segment"? "length" and "long" might be repetitive

**Review by JustinPeng13 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Yes!

File: docs/UserGuide.md (hunk 2 above)
Comment (bryanhce) [reply docs/UserGuide.md:186]: Changed!

**Review by JustinPeng13 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 3]
```diff
@@ -202,7 +202,8 @@ that you have inputted. The values that you did not specify to edit will remain
 
 ### Billing a guest : `bill`
 
-This command updates the bill of a guest in GuestBook.
+This command updates the bill of a guest in GuestBook. Note that the bill here refers to the additional costs incurred
+by the guest during his stay at your hotel and should not include the cost of the hostel stay.
```
Comment (JustinPeng13) [inline docs/UserGuide.md]: Could we phrase this as "Note that the bill here refers to the total additional cost incurred by the guest during their stay at your hotel and should not include the cost of the room booking."

**Review by JustinPeng13 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (bryanhce) [reply docs/UserGuide.md]: Fixed!

**Review by JustinPeng13 — APPROVED** [review]
lgtm

**Pull request merged**